experanto/data.py

```diff
@@ -36,25 +52,36 @@ def __getitem__(self, idx):
 
 class Mouse2pStaticImageDataset(Dataset):
     def __init__(
-            self, 
-            root_folder: str, 
-            tier: str, 
-            offset: float, 
-            stim_duration: float
-            ) -> None:
+        self,
+        root_folder: str,
+        tier: str,
+        offset: float,
+        stim_duration: float,
+        rescale: bool = False,
+        keep_nans: bool = False,
+        interpolation_mode: str = "linear",
+        interp_window: int = 5,
+    ) -> None:
         self.root_folder = Path(root_folder)
         self.tier = tier
         self.offset = offset
         self.stim_duration = stim_duration
-        self._experiment = Experiment(root_folder)
+        self._experiment = Experiment(
+            root_folder,
+            rescale=rescale,
+            keep_nans=keep_nans,
+            interpolation_mode=interpolation_mode,
+            interp_window=interp_window,
+        )
         self.device_names = self._experiment.device_names
         self.DataPoint = namedtuple("DataPoint", self.device_names)
```

Comment:
As related to #32, I'd suggest to not doing any datapoints and rather stick with dicts